toc/working-groups/docs.md

```diff
@@ -60,4 +60,27 @@ areas:
   - cloudfoundry/docs-credhub
   - cloudfoundry/docs-deploying-cf
   - cloudfoundry/docs-dotnet-core-tutorial
+- name: Cloud Foundry Tutorials
+  approvers:
+  - name: Steve Greenberg
+    github: spgreenberg
+  reviewers:
+  - name: Benjamin Guttmann
+    github: benjaminguttmann-avtq
+  - name: Maurice Brinkmann
+    github: mauricebrinkmann
+  - name: Andreas Koppenhöfer
+    github: akop
+  - name: Jovan Kostovski
+    github: chombium
```

Comment:
@beyhan , please add me so that:
1 - I can fix typos, etc.
2 - I can keep the info organization/architecture up-to-date
3 - I know when things are being changed so that I know when to publish.